clients/storage/blob.go

```diff
@@ -83,7 +95,7 @@ type ListContainersParameters struct {
 	Timeout    uint
 }
 
-func (p ListContainersParameters) GetParameters() url.Values {
+func (p ListContainersParameters) getParameters() url.Values {
```

Comment:
someone may yell at you for pulling this from the exported symbols, but I agree with you